### PR TITLE
[Acquisition team] Add buttons to navbar

### PR DESCRIPTION
**Pull request by akeller**
Follow up to https://github.com/camunda/camunda-docs-theme/pull/41 and adding the changes to this part of the docs site.

When running locally, two additional shortcode files were added, but they do not appear to be part of this work and are not in this PR.

### CODE REVIEW

File: themes/camunda/assets/css/docs.css
Comment (pepopowitz) [inline themes/camunda/assets/css/docs.css:1]: The large diffs on these assets (css and js) make me think they were not minified. Did you run the `optimize` command [described in the theme docs](https://github.com/camunda/camunda-docs-theme#consuming-the-theme)?

> 2) Build and update the theme in the target location by either running:
>    1) `grunt build` for building and syncing non-minified assets
>    2) Run additionally `grunt optimize` for building and syncing minified assets

**Review by pepopowitz — CHANGES_REQUESTED** [review]
I think the assets should be minified, otherwise this looks great!

File: themes/camunda/assets/css/docs.css
Comment (akeller) [reply themes/camunda/assets/css/docs.css:1]: Fixed, thank you!

**Review by tasso94 — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**